packages/@react-types/button/src/index.d.ts

```diff
@@ -29,25 +29,33 @@ export interface ButtonProps extends PressEvents, FocusableProps {
   target?: string
 }
 
-export interface SpectrumButtonProps extends ButtonProps, DOMProps, StyleProps {
+export interface AriaButtonProps extends ButtonProps, FocusableDOMProps, AriaLabelingProps {
```

Comment:
New pattern: an Aria type that extends the base type, and adds aria props and DOM props. The Spectrum type then extends the aria type.